sources/packages/api/src/route-controllers/assessment/assessment.aest.controller.ts

```diff
@@ -37,15 +37,17 @@ export class AssessmentAESTController extends BaseController {
    * pending and denied student appeal and scholastic
    * standings for an application.
    * @param applicationId, application number.
-   * @returns RequestAssessmentSummaryDTO list.
+   * @returns RequestAssessmentSummaryApiOutDTO list.
    */
   @Get("application/:applicationId/requests")
   @ApiOkResponse({
     description: "Requested assessment for the application found.",
+    type: RequestAssessmentSummaryApiOutDTO,
+    isArray: true,
```

Comment:
do we need this `ApiOkResponse`. as per our last discussion on API tags?

Reply:
My understanding is that we need it when it would add some valuable description.
If added, we need to add the type accordingly.

Reply:
I removed it since the comment was not adding much, IMO.